categories/README.md

```diff
@@ -2,20 +2,21 @@
 
 This folder is the home of the JSON file containing the info for the categories that are displayed on the website's home page. If any edits need to be made to these categories, just edit the data in the JSON and the home page will reflect your changes automatically.
 
-A repository can have up to 20 tags. Te list below also contains each category's blurb to ensure consistency in phrasing and length. Note that in some cases the category is plural but the corresponding tag is singular.
+A repository can have up to 20 tags. The list below also contains each category's blurb to ensure consistency in phrasing and length. Note that in some cases the category is plural but the corresponding tag is singular.
 
-For each category this file contains it's title, [icon][icon dir] filepath, and category description.
-
-This data is read by the [category info javascript][js dir].
+For each category this file contains it's title, [icon][icon dir] filepath, and category description. This data is read by the [category info javascript][js dir].
 
 [icon dir]: ../assets/images/
 [js dir]: ../js/category-info.js
 
-- Applications: Advance science with modeling and simulation codes - `application`
+- Applications: Advance computing with scientific simulations and IT system management codes - `application`
```

Comment:
"Advance" or "Advanced"?
Also, I think IT should be changed to HPC

Reply:
I was going for active verbs on the blurbs (advance, automate, browse, manage, etc.) Will change to HPC.